### PR TITLE
Added slash command for /fulledit

### PR DESCRIPTION
**Pull request by Fear-MK**
## Description

The title. I could not test this, slash commands wouldn't show.
Should work, but please test.

## Checklist

- [ ] If code changes were made, they have been tested
- [X] This PR fixes an issue
- [ ] This PR is not a code change (ie. documentation, typehinting, comments, etc.)

### CODE REVIEW

File: slash_cogs/TablingSlashCommands.py [hunk 1]
```diff
@@ -121,6 +121,20 @@ async def _edit_score(
...
         await commands.TablingCommands.change_player_score_command(message, this_bot, args, server_prefix, is_lounge)
+
+    @slash_command(name='fulledit',
+    description="Edit all players' score for a GP",
+    guild_ids=common.SLASH_GUILDS)
+    async def _edit_all_scores(
+        self,
+        ctx: discord.ApplicationContext,
+        gp: Option(int, "GP to edit"),
+        scores: Option(int, "New scores for players")
```
Comment (BadWolf1023) [inline slash_cogs/TablingSlashCommands.py:132]: This cannot work. You need to put in many numbers, so anything more than one number would be rejected.

**Comment by BadWolf1023** [comment]
@camelwater would be best person to review

File: slash_cogs/TablingSlashCommands.py (hunk 1 above)
Comment (Fear-MK) [reply slash_cogs/TablingSlashCommands.py:132]: text would work, right?

Comment (BadWolf1023) [reply slash_cogs/TablingSlashCommands.py:132]: > text would work, right?

Yes. Specifically, `str`

**Comment by Fear-MK** [comment]
Still can't test for some reason, but is this what you intended?

**Review by BadWolf1023 — APPROVED** [review]

**Pull request merged**